jib-core/src/main/java/com/google/cloud/tools/jib/image/Image.java

```diff
@@ -89,6 +90,18 @@
       return this;
     }
 
+    /**
+     * Sets the user name (or UID) and optionally the user group (or GID) which the process in the
```

Comment:
Wording suggestion: `Sets the user/group to run the container as.`

Reply:
done